src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -15,17 +14,60 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+    private static final String HELP_INTRO = "Below are some basic instructions to get started using PROperty!";
+    private static final String HELP_ADD_COMMAND = "Add Command \n" +
```

Comment:
For now the strings can be here, but in future it should be contained within each command itself so that we will not forget updating them when any of the commands change.